code/_globalvars/misc.dm

```diff
@@ -12,6 +12,8 @@ GLOBAL_LIST_INIT(duplicate_forbidden_vars, list("tag", "datum_components", "area
 	"group", "atmos_adjacent_turfs", "comp_lookup", "boxes", "click_border_start","storage_start", "storage_continue",\
 	"storage_end", "closer", "stored_start", "stored_continue", "stored_end", "click_border_end", "opened"))
 
+GLOBAL_LIST_INIT(ignored_atoms, list(/mob/dead, /obj/effect/landmark, /obj/docking_port, /obj/effect/particle_effect/sparks, /obj/effect/DPtarget, /obj/effect/supplypod_selector, /atom/movable/lighting_object))
```

Comment:
```suggestion
GLOBAL_LIST_INIT(ignored_atoms, typecacheof(list(/mob/dead, /obj/effect/landmark, /obj/docking_port, /obj/effect/particle_effect/sparks, /obj/effect/DPtarget, /obj/effect/supplypod_selector, /atom/movable/lighting_object))) 
```

Reply:
Won't this also apply to all subtypes of these, potentially being wasteful?

Reply:
it will replicate the behaviour of a long chain of istype() calls and is faster, if you dont want it to apply to subtypes theres an option you can supply to typecacheof(), `typecacheof(list(), FALSE, FALSE)`

Reply:
or just `typecacheof(list(), only_root_path=TRUE)`

Reply:
Even if you include the subtypes its still faster.

Reply:
Unless there's more than 10 million subtypes.

Reply:
log(10,000,000) = 7

Reply:
typecaches are FAST.